doc/manual/manual.tex

```diff
@@ -6949,9 +6949,124 @@ \subsubsection{The 2D cylindrical shell benchmarks by Davies et al.}
   \caption{\it Davies et al. benchmarks: Numerical results for some of the output quantities required by the benchmarks and the various cases considered.}
   \label{tab:davies-et-al-results}
 \end{table}
- 
 
 
+\subsubsection{ The Crameri et al. benchmarks}
+\label{sec:benchmark-crameri}
+
+\textit{This section was contributed by Ian Rose.}
+
+This section follows the two free surface benchmarks described by Crameri et al. \cite{CSG12}.  
+
+\paragraph{ Case 1: Relaxation of topography }
+\label{sec:benchmark-crameri-case-1}
+
+The first benchmark involves a high viscosity lid sitting on top of a lower viscosity 
+mantle. There is an initial sinusoidal topography which is then allowed to relax.
+This benchmark has a semi-analytical solution (which is exact for infinitesimally small
+topography). Details for the benchmark setup are in Figure~\ref{fig:crameri-benchmark-initial-topography}.
```

Comment:
Can you also link to the .prm file as is done in other cookbooks? You may want to excerpt the interesting parts to show readers how to set up such examples.